.config/dictionaries/project.dic

```diff
@@ -40,6 +41,7 @@ cborg
 cborseq
 cdrs
 cdylib
+
```

Comment:
empty line? can you please reformat this file? `just check-spelling` should do the job